benchmarks/time/src/main/scala/strawman/collection/immutable/ImmutableArrayBenchmark.scala

```diff
@@ -74,19 +96,40 @@ class ImmutableArrayBenchmark {
   }
 
   @Benchmark
-  def prependAll(bh: Blackhole): Unit = bh.consume(xs ++: xs)
+  @OperationsPerInvocation(1000)
+  def prependAll(bh: Blackhole): Unit = {
+    var ys = fresh(size)
+    val zs = fresh((size / 1000) max 1)
```

Comment:
We could initialize `ys` like so `var ys = xs`. For `zs`, I think it’s better to create the collection outside of the benchmark, with the `xss` thing that you deleted.